src/js/helpers/accountHelper.js

```diff
@@ -65,5 +49,45 @@ export const getSubmissionDeadlines = (fiscalYear, fiscalPeriod, submissionPerio
     return { submissionDueDate: submissionPeriod.submission_due_date, certificationDueDate: submissionPeriod.certification_due_date };
 };
 
-export const getLatestPeriodAsMoment = (availablePeriods) => getLatestPeriod(availablePeriods).asOfDate;
+export const getLatestPeriod = (availablePeriods, fy = null) => {
+    if (availablePeriods.length) {
+        return availablePeriods
+            .filter((s) => {
+                if (fy) {
+                    return s.submission_fiscal_year === parseInt(fy, 10);
+                }
+                return true;
+            })
+            .map((s) => ({
+                revealDate: moment.utc(s.submission_reveal_date),
+                asOfDate: moment.utc(s.period_end_date),
+                period: s.submission_fiscal_month,
+                year: s.submission_fiscal_year,
+                quarter: s.submission_fiscal_quarter
+            }))
+            .sort(({ revealDate: a }, { revealDate: b }) => b.valueOf() - a.valueOf())
+            .find(({ revealDate: s }) => moment(s).isSameOrBefore(moment()));
```

Comment:
This looks correct but I'm still seeing Nov 30th on the homepage - should it show Jan 8 2021? That is the most recent reveal date I see in redux